storage/objects/objects_test.go

```diff
@@ -173,6 +173,21 @@ func TestObjects(t *testing.T) {
 		t.Errorf("contents = %q; want %q", got, want)
 	}
 
+	t.Run("setMetadata", func(t *testing.T) {
+		bkt := client.Bucket(bucket)
+		obj := bkt.Object(object1)
+		err = setMetadata(ioutil.Discard, bucket, object1)
+		if err != nil {
+			t.Errorf("setMetadata: %v", err)
+		}
+		attrs, err := obj.Attrs(ctx)
```

Comment:
Maybe need to use `getMetadata` to check?
1. But if we'll have a problem with `getMetadata` then 2 tests will fail instead of one.
2. The one subtest for all metadata samples?

Reply:
No, `Attrs` seems okay.